### PR TITLE
yb/modify clang format style

### PR DESCRIPTION
**Pull request by yangbofun**
## Motivation and Context
* modify clang-format style (BreakAfterAttributes: Always)
If we don't set BreakAfterAttributes Always, the adaptor of DIOPI will work wrong.

before:
image

after:
image

## Checklist
**Before PR**:

- [x] I have read and followed the workflow indicated in the [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

**Review by jingguo-st — APPROVED** [review]

File: .clang-format [hunk 1]
```diff
@@ -24,6 +24,7 @@ Standard:  Cpp11
 AllowAllParametersOfDeclarationOnNextLine: true
 BinPackParameters: true
 BinPackArguments: false
+BreakAfterAttributes: Always
```
Comment (lljbash) [inline .clang-format:27]:
```suggestion
BreakAfterAttributes: Leave
```
考虑下这个，dipu 用的这个更灵活

**Review by lljbash — COMMENTED** [review]
/accelerate/commands/accelerate_cli.py

File: .clang-format (hunk 1 above)
Comment (yangbofun) [reply .clang-format:27]: > 另外是不是更应该考虑改 adaptor

1. always更直观，直接另起了一行。
2. adaptor后期会改，现在adaptor的那部分功能耦合很大，牵一发动全身。后续会重构。

Comment (yangbofun) [reply .clang-format]: > 另外是不是更应该考虑改 adaptor

这里耦合太多，一个函数，干了好几件事，后续会抽时间重构。

**Review by POI-WX — APPROVED** [review]

**Pull request merged**